### PR TITLE
Enabled scope os to support Red Hat

### PR DESCRIPTION
**Pull request by tgoettlicher**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 71c706642e9be6b1e65607f14eebad80965c4ee8 into 23f8d0bf3105e4507602b1ff020b7db0258181c3 )

File: spec/data/os/rhel6/etc/redhat-release [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Red Hat Enterprise Linux Server release 6.5 (Santiago)
+# This is a "SLES Expanded Support platform release 6.5"
```
Comment (thardeck) [inline spec/data/os/rhel6/etc/redhat-release]: Is this really representative?

Comment (tgoettlicher) [inline spec/data/os/rhel6/etc/redhat-release]: We can replace it by the file from the pennyworth vm.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged eb73940c90fdbe76f54333c9a168311b00e341f5 into 23f8d0bf3105e4507602b1ff020b7db0258181c3 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 560a8b9a93d6d2e7de4afc620f385bc0662d9cab into 23f8d0bf3105e4507602b1ff020b7db0258181c3 )

File: spec/unit/os_inspector_spec.rb [hunk 2]
```diff
@@ -67,6 +67,17 @@
...
+    it "gets os info from redhat release file" do
+      FakeFS::FileSystem.clone("spec/data/os/rhel6/etc/redhat-release",
+        "/etc/redhat-release")
+      os = inspector.send(:get_os_from_redhat_release, system)
+
+      expect(os.name).to eq "Red Hat Enterprise Linux Server"
+      expect(os.version).to eq "6.5 (Santiago)"
+    end
+  end
+
```
Comment (thardeck) [inline spec/unit/os_inspector_spec.rb]: I would remove the whole section since it is a private method and we are already testing it below.

Comment (tgoettlicher) [inline spec/unit/os_inspector_spec.rb]: Cornelius asked for a test for this method. See:
https://github.com/SUSE/machinery/pull/522/files#r22998839

**Comment by thardeck** [comment]
I am not a big fan of testing private methods but since it seems that Cornelius asked for it, lgtm.

**Pull request merged**